### PR TITLE
Add documentation about approval / validation requirements for DAO proposals

### PR DESCRIPTION
**Pull request by derekpierre**
…ting proposal..

**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [X] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [X] 2
- [ ] 3

**What this does:**
> High-level idea of the changes introduced in this PR. 
> List relevant API changes (if any), as well as related PRs and issues.

Documents requirements for > 60% support, and > 25% approval for DAO voting,.

**Issues fixed/closed:**
> - Fixes #...

**Why it's needed:**
> Explain how this PR fits in the greater context of the NuCypher Network.
> E.g., if this PR address a `nucypher/productdev` issue, let reviewers know!

**Notes for reviewers:**
> What should reviewers focus on? 
> Is there a particular commit/function/section of your PR that requires more attention from reviewers?

### CODE REVIEW

**Review by KPrasch — COMMENTED** [review]
Thanks for making the PR @derekpierre ! What do you think about describing in more detail voter qualification, the ERT, how to cast votes, and linking to some Aragon resources?

**Comment by derekpierre** [comment]
> Thanks for making the PR @derekpierre ! What do you think about describing in more detail voter qualification, the ERT, and how to cast votes, and linking to some Aragon resources?

Regarding the ERT, I can add some of the information provided here - https://dao.nucypher.com/t/emergency-response-team/28. 

How about we do a more comprehensive update as a follow-up PR - filed #2601.

**Review by KPrasch — APPROVED** [review]
Yeah, Okay.  Happy to see *something* in this section of the docs.

File: docs/source/architecture/dao.rst [hunk 1]
```diff
@@ -31,11 +31,18 @@ proposal to the community, all stakers have the chance to validate that proposal
 .. image:: ../.static/img/dao_proposal.png
     :target: ../.static/img/dao_proposal.png
 
-Clicking on ``Yes`` or ``No`` triggers an Ethereum transaction signalling each staker's opinion on the proposal.
-Depending on whether a quorum is reached, proposals may be validated rejected. If they validated, they can take
-immediate effect on the NuCypher Network via the Agent.
+Clicking on ``Yes`` or ``No`` triggers an Ethereum transaction signalling each staker's opinion on the proposal. That
+staker's opinion is weighted by their stake i.e. if they have 30,000 NU staked, then voting ``Yes`` or ``No`` will have
+a weight of 30,000 NU tokens.
```
Comment (cygnusv) [inline docs/source/architecture/dao.rst:36]:
```suggestion
staker's opinion is weighted by their stake i.e. if they have 30,000 NU staked, then that staker will have
a weight of 30,000 NU tokens.
```

File: docs/source/architecture/dao.rst [hunk 2]
```diff
@@ -31,11 +31,18 @@ proposal to the community, all stakers have the chance to validate that proposal
...
 
-Note that not all proposals necessarily aim to produce on-chain effects on the network's smart contracts; some
+Depending on whether a quorum is reached, proposals may be validated or rejected. A proposal can only be validated
+if and only if:
+
+#. it receives > 60% support i.e. of all the tokens that voted, more than 60% are required to vote ``Yes``
+#. it receives > 25% approval i.e. more than 25% of all the staked tokens that could have possibly voted are required to vote ``Yes``
```
Comment (cygnusv) [inline docs/source/architecture/dao.rst:42]:
```suggestion
#. it receives > 60% support, i.e., of all the tokens that participated, more than 60% are required to signal ``Yes``
#. it receives > 25% approval, i.e., more than 25% of all the staked tokens that could have possibly participated are required to signal ``Yes``
```

**Review by cygnusv — APPROVED** [review]

**Pull request merged**